src/config/config.service.ts

```diff
@@ -108,4 +109,8 @@ export class ConfigService {
   public getDeadLetterPrefix(): string {
     return this.nestConfigService.get<string>('DEAD_LETTER_JOB_PREFIX')!;
   }
+
+  public getPageSize(): number {
+    return this.nestConfigService.get<number>('CONNECTIONS_PER_PROVIDER_RESPONSE_PAGE')!;
```

Comment:
I would test this. The NestJS config service has what I consider to be a bug--even though we specify `<number>` and return type `number`, because this comes from an environment string, the actual returned value is `"100"` instead of `100`. In the past I've had to wrap the return value in `Number(...)` to force the correct type.

Reply:
Hmm isnt `Joi` intercepts this before? https://github.com/AmplicaLabs/reconnection-service/blob/main/src/config/env.config.ts

Reply:
Joi only validates the environment, it doesn't change the type. I got bitten by this in Community Rewards. Just log the `typeof` the value you get back from the config service and see if it's `string` or `number`

Reply:
@saraswatpuneet Can you go ahead and wrap it? I've seen the same issue elsewhere.

Then we can get this merged.

Reply:
@JoeCap08055  just tested by logging

` this.logger.error('pageSize', typeof pageSize)`

here is the output
```
 ERROR [ReconnectionGraphService] pageSize
number

```

So looks like it is number as expected

Reply:
Changed the env to page size 1000 , to test without default 100 number, same result, the number is read correctly from env 

```
pageSize 1000
[Nest] 2020930  - 04/10/2024, 12:27:45 PM   ERROR [ReconnectionGraphService] pageSize
number
```

Reply:
What if you set the env to `"1000"`?

Reply:
Same, it get parsed as number, also tried with "2000" 

```
pageSize 2000
[Nest] 2061578  - 04/10/2024, 12:59:00 PM   ERROR [ReconnectionGraphService] pageSize
number


```
![image](https://github.com/AmplicaLabs/reconnection-service/assets/61435908/6d18d608-e2bd-44d9-bcbc-2e0c59a81382)